draft-irtf-cfrg-hpke.md

```diff
@@ -710,11 +715,54 @@ byte strings for public keys.
 | 0x0002 | AES-GCM-256      | 32  | 12  | {{GCM}}      |
 | 0x0003 | ChaCha20Poly1305 | 32  | 12  | {{?RFC8439}} |
 
-# Security Considerations
-
-The general security properties of HPKE are described in
-{{security-properties}}.  In this section, we consider a security issue that may
-arise in practice and an advanced use case.
+# Security Considerations {#sec-considerations}
+
+## Security Properties {#sec-properties}
+
+HPKE makes the following cryptographic assumptions about KDF interfaces:
+
+- Hash: Collision resistance.
+- Extract: Indifferentiable from a random oracle.
+- Expand: Behaves as a pseudorandom function wherein the first
+argument is the key.
+
+The functions specified in {{kdf-ids}} achieve these properties.
+Moreover, for DH-based KEMs, HPKE assumes the gap Computational Diffie-Hellman
+(CDH) problem is hard {{GAP}}. The DH-based KEMs specified in {{kem-ids}}
+also achieve these properties.
```

Comment:
Nit: "Moreover" is unnecessary.  

You're also still missing what the general KEM property is.  I would arrange as 

* General KEM property
* Probably a paragraph break
* DH-KEM satisfies the general KEM property if gap DH
* KDFs and AEAD functions in this doc meet the requirements

Reply:
We can't say anything about general KEMs yet. The analysis does not cover that!